### PR TITLE
MAUI-900341-Prepare-UG-for-Vol3-features

### PR DESCRIPTION
**Pull request by HadayaRahman-SF4673**
Added UG Documentation for the following features
-Navigation to Adjacent month in SfCalendar
-Tapped Event Support for Week Numbers in SfCalendar
-Minimum and Maximum Support for SfTimePicker

Related Task : https://dev.azure.com/EssentialStudio/Mobile%20and%20Desktop/_workitems/edit/900341/

### CODE REVIEW

File: MAUI/TimePicker/time-restriction.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+platform: maui
+control: SfTimePicker
+documentation: ug
+---  
+
+# Time restriction in .NET MAUI Time Picker (SfTimePicker)
+
+## Minimum Time
+
+The time picker provides an option to restrict the selection of time items by using the [MinimumTime]() property in [SfTimePicker](), and you cannot select the times beyond the minimum time range. The `MinimumTime` value has to be lesser than the `MaximumTime` value.When you set minimum time as timespan only hour and minute gets considered.
```
[the remark this answers is not in the document]
Comment (MuniappanSubramanian) [reply MAUI/TimePicker/time-restriction.md:14]: Like wise update maximum time

File: maui-toc.html [hunk 2]
```diff
@@ -1289,6 +1289,7 @@
             <li><a href="/maui/TimePicker/events">Events</a></li>
             <li><a href="/maui/TimePicker/localization">Localization</a></li>
             <li><a href="/maui/TimePicker/accessibility">Accessibility</a></li>
+            <li><a href="/maui/TimePicker/time-restriction">Time Restriction</a></li>
```
Comment (MuniappanSubramanian) [inline maui-toc.html:1292]: Add after TextDisplayMode

File: MAUI/Calendar/date-navigation.md [hunk 3]
```diff
@@ -70,6 +70,29 @@ this.Calendar.AllowViewNavigation = true;
 {% endhighlight %}
 {% endtabs %}
 
+## Navigation to Adjacent Months Using Leading/Trailing Dates
+You can navigate to the next or previous month by tapping on the leading or trailing dates within the Calendar control. When you tap on a leading date, the calendar will navigate to the previous month, tapping on a trailing date will navigate to the next month. In `SfCalendar`, this functionality can be enabled or disabled by modifying the [NavigateToAdjacentMonth]() property.
```
Comment (MuniappanSubramanian) [inline MAUI/Calendar/date-navigation.md:74]: Here's the revised content:

---

You can navigate to the next or previous month in the Calendar control by tapping on the leading or trailing dates. Tapping on a leading date moves the calendar to the previous month, while tapping on a trailing date moves it to the next month. In SfCalendar, this functionality can be enabled or disabled using the `[NavigateToAdjacentMonth]()` property.

File: MAUI/Calendar/date-navigation.md [hunk 4]
```diff
@@ -70,6 +70,29 @@ this.Calendar.AllowViewNavigation = true;
 {% endhighlight %}
 {% endtabs %}
 
+## Navigation to Adjacent Months Using Leading/Trailing Dates
```
Comment (MuniappanSubramanian) [inline MAUI/Calendar/date-navigation.md:73]: Navigating Adjacent Months with Leading and Trailing Dates

File: MAUI/TimePicker/time-restriction.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (HadayaRahman-SF4673) [reply MAUI/TimePicker/time-restriction.md:14]: Updated the both minimum and maximum time content

File: maui-toc.html (hunk 2 above)
Comment (HadayaRahman-SF4673) [reply maui-toc.html:1292]: Added time restriction after text display mode

File: MAUI/Calendar/date-navigation.md (hunk 3 above)
Comment (HadayaRahman-SF4673) [reply MAUI/Calendar/date-navigation.md:74]: Updated the description

File: MAUI/Calendar/date-navigation.md (hunk 4 above)
Comment (HadayaRahman-SF4673) [reply MAUI/Calendar/date-navigation.md:73]: Changed it

File: MAUI/Calendar/date-navigation.md [hunk 5]
```diff
@@ -70,6 +70,29 @@ this.Calendar.AllowViewNavigation = true;
...
+
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+
+this.Calendar.NavigateToAdjacentMonth = true;
+
+{% endhighlight %}
+{% endtabs %}
+
+![Month view navigation to adjacent months using leading/trailing dates in .NET MAUI Calendar.](images/date-navigation/maui-month-view-navigation-by-leading-trailing-month-dates.gif)
```
[the remark this answers is not in the document]
Comment (HadayaRahman-SF4673) [reply MAUI/Calendar/date-navigation.md:94]: Okay

**Review by MuniappanSubramanian — APPROVED** [review]

**Pull request merged**